modules/ROOT/pages/salesforce/salesforce-connector-100.adoc

```diff
@@ -0,0 +1,204 @@
+= Salesforce Connector 10.0 - Get Started - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Salesforce Connector v10.0
+
+Anypoint Connector for Salesforce (Salesforce Connector) enables you to create apps that react to Salesforce events such as adding, changing, or deleting objects, topics, documents, and channels.
+
+Salesforce Connector enables you to connect to the Salesforce APIs. This connector exposes methods for accessing Salesforce, including working with the https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_dev_process_chapter.htm[Salesforce Apex classes].
+
+This connector works with the Salesforce SOAP API, REST API, Bulk API, and Streaming API, depending on the operation you configure. Each API call uses a request/response pattern over an HTTPS connection. All required request headers, error handling, and HTTPS connection configurations are built into the connector.
+
+== About Connectors
+
+Anypoint Connectors are Mule runtime engine extensions that enable you to connect to APIs and resources on external systems, such as Salesforce, ServiceNow, and Twitter.
+
+== Prerequisites
+
+Before creating an app, you must have access to the Salesforce target resource and Anypoint Platform. You must also understand how to create a Mule app using Design Center or Anypoint Studio, and be familiar with Salesforce, HTTP, SOAP, REST, Bulk and Streaming APIs, Mule runtime engine (Mule), Anypoint Connectors, Anypoint Studio (Studio), Mule concepts, elements in a Mule flow, and Global Elements.
+
+Requirements for using the connector:
+
+* https://developer.salesforce.com[Salesforce developer account]
+* https://help.salesforce.com/articleView?id=user_security_token.htm[Salesforce security token]
++
+You can receive a new security token by email if you run *Reset Security Token* through *My Personal Information* from the Salesforce Setup pages.
+* Consumer key and Secret (available in your Salesforce developer account)
++
+Required if you are using the OAuth Authentication type.
++
+* Namespace and schema location
++
+If you plan to create the XML for your Mule app, you need to include the correct namespace and schema location in your XML file. Anypoint Studio adds this information to the XML file automatically when you add the Salesforce connector to a flow in a Mule app.
```

Comment:
If you plan to create the XML for your Mule app, include the correct namespace and schema location in your ....

(omit "you need to")